client/actions/actions.js

```diff
@@ -0,0 +1,137 @@
+import * as ActionTypes from './actionTypes';
+import { asyncAction } from '../middleware/asyncMiddleware';
+import axios from 'axios';
+
+/**
+ * Action Creators
+ */
+
```

Comment:
For the most part, there are three actions for each http request you can make (CRUD)
example: Create has an action for starting the request(CREATE_USER), finishing the request successfully (CREATE_USER_SUCCESS), and if it fails (CREATE_USER_FAILURE). the application's state is updated accordingly in each of these cases